system/modules/core/templates/jquery/j_mediaelement.html5

```diff
@@ -9,7 +9,12 @@ $GLOBALS['TL_CSS'][] = 'assets/jquery/mediaelement/'. $GLOBALS['TL_ASSETS']['MED
 <script>
   (function($) {
     $(document).ready(function() {
-      $('video,audio').mediaelementplayer({
+
+      var media_items = $('video').filter(function() {
+        return $(this).parent().prop("tagName") !== "PICTURE";
```

Comment:
Shouldn't this be `.tagName.toLowerCase() !== 'picture'`? I don't think we need to use `prop()` here.